### PR TITLE
Test transform

### PR DESCRIPTION
**Pull request by BenGriffith**
Minor updates to project structure and started writing unit tests for transform module helper methods

### CODE REVIEW

File: tests/unit/test_transform_helpers.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        {"month_key": 9, "name": "September"},
+        {"month_key": 10, "name": "October"},
+        {"month_key": 11, "name": "November"},
+        {"month_key": 12, "name": "December"},
+    ]
+
+
+def test_date_dim_row(transform, date_dim_rows):
+    rows = transform.date_dim_row()
+    rows == [date_dim_rows]
```
Comment (bbelderbos) [inline tests/unit/test_transform_helpers.py:34]: missingassert?

File: tests/conftest.py [hunk 2]
```diff
@@ -76,4 +81,143 @@ def mock_blob():
...
+    return Transform(
+        storage_client=mock_gcs_client,
+        bq_client=mock_bq_client,
+        bucket_name="project-cryptocurrency"
+    )
+
+
+@pytest.fixture
+def date_dim_rows():
+    today = datetime.today()
```
Comment (bbelderbos) [inline tests/conftest.py:98]: do we need to "freeze" this for testing?

**Review by bbelderbos — COMMENTED** [review]
Looks good

File: tests/conftest.py (hunk 2 above)
Comment (bbelderbos) [reply tests/conftest.py:98]: ok maybe I need to  look  at the date_dim_row implementation ...

yep in that case the recommended lib is freezegun, related article: https://pybit.es/articles/guest-freezegun

Comment (BenGriffith) [reply tests/conftest.py:98]: Ok, cool. I'll work on adding it.

**Pull request merged**